community/mumble/APKBUILD

```
@@ -2,66 +2,51 @@
 # Contributor: Johannes Matheis <jomat+alpinebuild@jmt.gr>
 # Maintainer: Johannes Matheis <jomat+alpinebuild@jmt.gr>
 pkgname=mumble
-pkgver=1.2.19
-pkgrel=9
+pkgver=1.3.0_rc1
+_pkgver=${pkgver/_/-}
+__pkgver=${pkgver/_rc1/}
```

This will change whenever rc2 comes out.
Perhaps use wildcards instead?
```suggestion
__pkgver=${pkgver/_rc*/}
```